tests/python/gaia-ui-tests/gaiatest/apps/email/app.py

```diff
@@ -142,7 +143,7 @@ def a11y_navigate_to_manual_setup(self, name, email):
 
     @property
     def header(self):
-        return Header(self.marionette)
+        return Header(self.marionette, self.marionette.find_element(*self.header_locator))
```

Comment:
Actually, leave it as it is. We need this, as PageRegion takes the root element as parameter.